pavelib/assets.py

```diff
@@ -64,6 +64,20 @@
     'requirejs/require.js',
     'underscore.string/dist/underscore.string.js',
     'underscore/underscore.js',
+    '@edx/studio-frontend/dist/manifest.min.js',
+    '@edx/studio-frontend/dist/manifest.min.js.map',
+    '@edx/studio-frontend/dist/vendor.min.js',
+    '@edx/studio-frontend/dist/vendor.min.js.map',
+    '@edx/studio-frontend/dist/vendor.min.css',
+    '@edx/studio-frontend/dist/vendor.min.css.map',
+    '@edx/studio-frontend/dist/assets.min.js',
+    '@edx/studio-frontend/dist/assets.min.js.map',
+    '@edx/studio-frontend/dist/assets.min.css',
+    '@edx/studio-frontend/dist/assets.min.css.map',
+    '@edx/studio-frontend/dist/accessibilityPolicy.min.js',
+    '@edx/studio-frontend/dist/accessibilityPolicy.min.js.map',
+    '@edx/studio-frontend/dist/accessibilityPolicy.min.css',
+    '@edx/studio-frontend/dist/accessibilityPolicy.min.css.map',
```

Comment:
Are we at all concerned about this list growing much larger as more and more SFE pages are written?

Reply:
Yes, this will likely grow much larger :( . Our pavelib `process_npm_assets` function isn't smart enough to process a whole directory. Though, it might not be too hard to add that functionality... I can try that.